src/main/java/adris/altoclef/ConfigState.java

```diff
@@ -332,11 +350,11 @@ private void applyState(Settings s, AltoClefSettings sa) {
                 }
             }
 
-            sa.setFlowingWaterPass(_allowWalkThroughFlowingWater);
+            sa.setFlowingWaterPass(allowWalkThroughFlowingWater);
             sa.allowShears(allowShears);
 
             // Extra / hard coded
-            RayTraceUtils.fluidHandling = rayFluidHandling;
+            RayTraceUtils.fluidHandling = rayFluidHandling; // ?
```

Comment:
When raytracing, you can chose how liquids are handled (either they are ignored, or included, or just the source/solid blocks are included or something like that). Tasks may want to change that behavior depending on what they're doing.

But now that I think about it, this might be a bad place to add that. This "fluidHandling" should probably be passed to the raytrace calls, and not set here. Might be an issue.